### ISSUE TITLE
loosen dependency requirements

### ISSUE THREAD

**Issue by dereekb** [tanansatpal/ngx-mat-intl-tel-input#149]
Because of some of the recent npm changes regarding peer dependencies, I'm running into dependency conflicts with this library when running npm install:

```
npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/dereekb/.npm/_logs/2022-03-02T04_52_54_822Z-debug-0.log
dereekb@dbMBP dbcomponents % npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: @dereekb/components@1.0.0
npm ERR! Found: @angular/common@13.2.4
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"^13.0.0" from the root project
npm ERR!   peer @angular/common@"^13.0.0 || ^14.0.0-0" from @angular/cdk@13.2.4
npm ERR!   node_modules/@angular/cdk
npm ERR!     @angular/cdk@"^13.0.0" from the root project
npm ERR!     peer @angular/cdk@"^13.0.0" from @angular/flex-layout@13.0.0-beta.38
npm ERR!     node_modules/@angular/flex-layout
npm ERR!       @angular/flex-layout@"^13.0.0-beta.38" from the root project
npm ERR!     2 more (@angular/material, ng-overlay-container)
npm ERR!   13 more (@angular/fire, @angular/platform-browser, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! ngx-mat-intl-tel-input@"^4.1.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: @angular/common@13.0.3
npm ERR! node_modules/@angular/common
npm ERR!   peer @angular/common@"~13.0.0" from ngx-mat-intl-tel-input@4.1.0
npm ERR!   node_modules/ngx-mat-intl-tel-input
npm ERR!     ngx-mat-intl-tel-input@"^4.1.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

All the angular dependencies on this project should be safe to change from ~13.0.0 to ^13.0.0, which would allow any minor version rather than any patch version.

For now I can work around with `npm install --legacy-peer-deps`. I would make a PR, but I just checked and there's one already available, #146.

**Comment by rbalet** [comment]
Hi @dereekb>

I did it in the [clone library](https://github.com/rbalet/ngx-mat-input-tel).
Sadly, I do only accept version from the `v15`, hopes this isn't a problem for you

![image](https://github.com/user-attachments/assets/603fd276-619a-4fba-9a90-edc619a058f2)

**Comment by dereekb** [comment]
Thanks! This change did get merged in for the v5.0.0 release of this library so it has been working for me.

I'll be looking into your clone though, thanks for working on a fork!
